README.md

```diff
@@ -1,8 +1,36 @@
-# Serverless Framework: Deploy on Scaleway Functions
+# Scaleway Plugin for Serverless Framework
 
-The Scaleway functions plugin for [Serverless Framework](https://serverless.com/) allows users to deploy their functions and containers to Scaleway Functions with a simple `serverless deploy`.
+This is the [Scaleway Functions](https://www.scaleway.com/en/serverless-functions/) plugin for [Serverless Framework](https://serverless.com/).
 
-Serverless Framework handles everything from creating namespaces to function/code deployment by calling APIs endpoint under the hood.	
+## Quick-start
```

Comment:
I think we should update our documentation too or just keep a link to our doc : https://www.scaleway.com/en/docs/compute/functions/api-cli/fun-uploading-with-serverless-framework/

Reply:
Yes very good point. I think that keeping the docs with the code (i.e. in Github) is the best way to keep them up to date. It means creating a link from those Scaleway docs, so I'll do that once this is merged.

Reply:
We also have this: https://www.scaleway.com/en/docs/compute/containers/api-cli/cont-uploading-with-serverless-framework/

Reply:
And this: https://www.scaleway.com/en/docs/tutorials/install-serverless-framework/